### PR TITLE
added changes for issue-50

### PR DESCRIPTION
**Pull request by cnduc**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/HakinCodes/Malaria-Detection/blob/master/CONTRIBUTING.md)?

Yes

### Description

Documentation that can help us form our Wiki later using the EDA Notebook.

### Checklist

- [x] I've read the contribution guidelines.
- [x] This PR mets all the requirements like Clean Code, proper README and Documentation

### Related Issues or Pull Requests

 Fixes: #50 

### Screenshots (if any)

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot** | **updated screenshot**|

### CODE REVIEW

File: Documentation/malaria_documentation_summary .md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+<h3>RISK FACTOR</h3>
+
+<p>Malaria is commonly associated with poverty and has a major negative effect on economic development. In Africa, it is estimated to result in losses of <b>US$12 billion</b> a year due to increased healthcare costs, lost ability to work, and negative effects on tourism</p>
+
+<h3>SYMPTOMS OF MALARIA</h3>
+
+<p>A malaria infection is generally characterized by the following signs and symptoms:</p><br>
```
Comment (bhadreshpsavani) [inline Documentation/malaria_documentation_summary .md:13]: We can remove the article here
```
<p>Malaria infection is generally characterized by the following signs and symptoms:</p><br>
```

File: Documentation/malaria_documentation_summary .md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+<h3>RISK FACTOR</h3>
+
+<p>Malaria is commonly associated with poverty and has a major negative effect on economic development. In Africa, it is estimated to result in losses of <b>US$12 billion</b> a year due to increased healthcare costs, lost ability to work, and negative effects on tourism</p>
```
Comment (bhadreshpsavani) [inline Documentation/malaria_documentation_summary .md:9]: We need to add the article here
```
</b> a year due to increased healthcare costs, lost the ability to work, and negative effects on tourism</p>
```

File: Documentation/malaria_documentation_summary .md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+<h3>SYMPTOMS OF MALARIA</h3>
+
+<p>A malaria infection is generally characterized by the following signs and symptoms:</p><br>
+<ul><li>Fever</li><li>Chills</li><li>Headache</li><li>Nausea and vomiting</li><li>Muscle pain and fatigue</li></ul><p>Other signs and symptoms may include:</p><ul><li>Sweating</li><li>Chest or abdominal pain</li><li>Cough</li></ul>
+
+<h3>How is Malaria diagnosed by pathologists?</h3>
+
+<p>Typically Malaria is diagnosed by microscopic examination of blood cells under the supervision of a pathologist. Red blood cells are examined using a microscope using blood films. The pathologists tries to find evidence of Malaria using past domain knowledge. Typically, when a cell is infected with Malaria one can see distorted cell shapes which are also accompanied by certain blunt spots in the cell.</p>
```
Comment (bhadreshpsavani) [inline Documentation/malaria_documentation_summary .md:18]: Reframe
```
pathologists tries to find evidence
```
As
```
pathologists try to find evidence
```

File: Documentation/malaria_documentation_summary .md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+<h3>PROBLEM STATEMENT</h3>
+
+<p>For malaria as well as other microbial infections, manual inspection of thick and thin blood smears by trained microscopists remains the gold standard for parasite detection and stage determination because of its low reagent and instrument cost and high flexibility. Despite manual inspection being extremely low throughout and susceptible to human bias, automatic counting software remains largely unused because of the wide range of variations in brightfield microscopy images. However, a robust automatic counting and cell classification solution would provide enormous benefits due to faster and more accurate quantitative results without human variability; researchers and medical professionals could better characterize stage-specific drug targets and better quantify patient reactions to drugs.</p>
+
+<h2>General Outline</h2>
+
+<p><b>Our code template shall perform the following steps:</b></p>
+
+<ol><li>Importing Libraries</li><li>Preliminary Data Processing.</li><li>Check total number of entries</li><li>Exploratory Data Analysis (EDA).</li></ol>
```
Comment (bhadreshpsavani) [inline Documentation/malaria_documentation_summary .md:28]: Add the article here
```
<li>Check total number of entries</li>
```
Like this
```
<li>Check the total number of entries</li>
```

File: Documentation/malaria_documentation_summary .md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+<h3>Information about the dataset.</h3>
+
+<p>The dataset is downloaded from Kaggle from this link — <a href="https://www.kaggle.com/iarunava/cell-images-for-detecting-malaria">https://www.kaggle.com/iarunava/cell-images-for-detecting-malaria</a>. The dataset was originally taken from the NIH website and uploaded to a Kaggle repository. The dataset contains 27558 cell images. Out of these 27558 images, we have 13779 cell images which are infected by Malaria and an other 13779 images which are uninfected.
```
Comment (bhadreshpsavani) [inline Documentation/malaria_documentation_summary .md:32]: Reframe the last sentence like this,
```
we have 13779 cell images that are infected by Malaria and another 13779 uninfected images.
```

**Review by bhadreshpsavani — CHANGES_REQUESTED** [review]

**Comment by bhadreshpsavani** [comment]
Hi @cnduc,

I have requested a few grammatical changes for the file

File: Documentation/malaria_documentation_summary .md (hunk 2 above)
Comment (cnduc) [reply Documentation/malaria_documentation_summary .md:9]: @bhadreshpsavani noted, will make changes.

File: Documentation/malaria_documentation_summary .md (hunk 3 above)
Comment (cnduc) [reply Documentation/malaria_documentation_summary .md:18]: @bhadreshpsavani sure, will do.

File: Documentation/malaria_documentation_summary .md (hunk 4 above)
Comment (cnduc) [reply Documentation/malaria_documentation_summary .md:28]: @bhadreshpsavani will do.

File: Documentation/malaria_documentation_summary .md (hunk 5 above)
Comment (cnduc) [reply Documentation/malaria_documentation_summary .md:32]: @bhadreshpsavani sure, will do.

**Comment by cnduc** [comment]
@bhadreshpsavani working on this.

**Review by bhadreshpsavani — APPROVED** [review]

**Comment by bhadreshpsavani** [comment]
Hi @BALaka-18 @macabdul9 and @sarthakkundra,
Can you also check these changes?

**Review by macabdul9 — APPROVED** [review]

**Pull request merged**